dashboard/app/models/pd/workshop.rb

```diff
@@ -347,6 +351,16 @@ def year
     sessions.order(:start).first.start.strftime('%Y')
   end
 
+  # Suppress 3 and 10-day reminders for certain workshops
+  def suppress_reminders?
```

Comment:
I can't decide if I would prefer to have a `should_send_reminders?` so that a positive return value reflects a positive action, or keep it like this so that it's clearer that suppression is the exception. Thoughts?